impl/client/src/main/javascript/web/pentaho/visual/ccc/barAbstract/model.js

```diff
@@ -18,24 +18,25 @@ define([
   "../categoricalContinuousAbstract/model",
   "pentaho/i18n!../abstract/i18n/model",
   "../abstract/mixins/settingsMultiChartType"
-], function(module, categoricalContinuousAbstractModelFactory, bundle, settingsMultiChartType) {
+], function(module, baseModelFactory, bundle, settingsMultiChartType) {
```

Comment:
models: `pentaho/visual/common/bar` and views: `pentaho/visual/ccc/bar` ?
Until we have a more powerful service registry, though, we'd need to still configure the `defaultView` of  `pentaho/visual/common/bar` to `pentaho/visual/ccc/common/bar`.